src/js/components/Layout/Footer.jsx

```diff
@@ -64,15 +102,22 @@ const mapStateToProps = state => ({
   timezone: state.session.timezone,
   ipAddress: state.session.ipAddress,
   languages: getLanguages(state.localize),
+  localizationModeEnabled: state.session.localizationModeEnabled,
+  translationModeLocale: state.session.translationModeLocale,
 });
 
-export default connect(mapStateToProps, { changeCurrentLocale, setActiveLanguage })(Footer);
+const mapDispatchToProps = {
+  changeLocale: changeCurrentLocale,
+  setLanguage: setActiveLanguage,
```

Comment:
What do you think about using `useDispatch()` instead of `mapDispatchToProps`? I found an interesting discussion on stackoverflow about this two approach [Should I use useselector/useDispatch instead of mapStateToProps](https://stackoverflow.com/questions/59748180/should-i-use-useselector-usedispatch-instead-of-mapstatetoprops) and i am curious what do you think

Reply:
I don't see any pros/cons of using one versus another. I think it's rather what you prefer and what is more useful in each scenario (for example you can't use `mapDispatchToProps` inside a custom hook, so you are obligated to use `useDispatch()`. 
What I'm sure about is that we shouldn't mix both if possible and probably in "regular" components stick with `connect` approach as we've done for whole project, as it might be easier to read and debug, because we are used to this convention.
@drodzewicz any additional thoughts?